Make the library configuration universal for 32/64bits.

Solaris specific. Not suitable for upstream.

--- libxml2-2.14.6/libxml-2.0.pc.in
+++ libxml2-2.14.6/libxml-2.0.pc.in
@@ -8,5 +8,5 @@ Name: libXML
 Version: @VERSION@
 Description: libXML library version2.
 Requires@XML_PC_PRIVATE@: @XML_PC_REQUIRES@
-Libs: @XML_LIBDIR@ @XML_LIBS@ @XML_PC_LIBS_PRIVATE@ @XML_PC_LIBS@ @LIBS@
+Libs: @XML_LIBS@ @XML_PC_LIBS_PRIVATE@ @XML_PC_LIBS@ @LIBS@
 Cflags: @XML_INCLUDEDIR@ @XML_PC_CFLAGS_PRIVATE@ @XML_STATIC_CFLAGS@
--- libxml2-2.14.6/xml2-config.in
+++ libxml2-2.14.6/xml2-config.in
@@ -82,16 +82,7 @@ echo @WITH_MODULES@
        	;;
 
     --libs)
-        if [ "$2" = "--dynamic" ]; then
-            shift
-            libs="@XML_LIBS@ @XML_PRIVATE_LIBS_NO_SHARED@"
-        else
-            libs="@XML_LIBS@ @XML_PRIVATE_LIBS@ @LIBS@"
-        fi
-
-        if [ "@libdir@" != "/usr/lib" -a "@libdir@" != "/usr/lib64" ]; then
-            libs="@XML_LIBDIR@ $libs"
-        fi
+        libs="@XML_LIBS@"
         ;;
 
     *)
